### PR TITLE
Add aten::lshift/rshift/prelu and thieir variants   

### PR DESCRIPTION
**Pull request by yucai-intel**
- [x] prelu
- [x] __lshift__
- [x] __ilshift__
- [x] __rshift__
- [x] __irshift__

### CODE REVIEW

File: yaml/xpu_functions.yaml [hunk 1]
```diff
@@ -127,6 +127,17 @@ supported:
   - relu
   - relu_
   - relu.out
+  - prelu
```
Comment (majing921201) [inline yaml/xpu_functions.yaml:130]: no need to register, CUDA doesn't

File: src/ATen/native/xpu/Activation.cpp [hunk 2]
```diff
@@ -633,6 +634,78 @@ Tensor& XPUNativeFunctions::softshrink_backward_out(
   return grad_input;
 }
 
+Tensor XPUNativeFunctions::prelu(const Tensor& self, const Tensor& weight_) {
```
Comment (majing921201) [inline src/ATen/native/xpu/Activation.cpp:637]: remove

File: src/ATen/native/xpu/sycl/ActivationPreluKernels.cpp [hunk 3]
```diff
@@ -0,0 +1,48 @@
+#include <ATen/ATen.h>
+#include <ATen/Dispatch.h>
+#include <ATen/NumericUtils.h>
+#include <ATen/core/TensorBase.h>
+#include <ATen/native/Activation.h>
```
Comment (fengyuan14) [inline src/ATen/native/xpu/sycl/ActivationPreluKernels.cpp:5]: Please remove the include if no need.

File: src/ATen/native/xpu/sycl/ActivationPreluKernels.cpp [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+#include <ATen/native/TensorIterator.h>
+#include <c10/core/Scalar.h>
+
+#include <ATen/native/xpu/sycl/Loops.h>
+#include <comm/XPUMathCompat.h>
```
Comment (fengyuan14) [inline src/ATen/native/xpu/sycl/ActivationPreluKernels.cpp:10]: Please remove the include if no need.

File: src/ATen/native/xpu/sycl/ActivationPreluKernels.cpp [hunk 5]
```diff
@@ -0,0 +1,48 @@
+#include <ATen/ATen.h>
+#include <ATen/Dispatch.h>
+#include <ATen/NumericUtils.h>
```
Comment (fengyuan14) [inline src/ATen/native/xpu/sycl/ActivationPreluKernels.cpp:3]: Please remove the include if no need.

File: src/ATen/native/xpu/sycl/ActivationPreluKernels.cpp [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+#include <ATen/core/TensorBase.h>
+#include <ATen/native/Activation.h>
+#include <ATen/native/TensorIterator.h>
+#include <c10/core/Scalar.h>
```
Comment (fengyuan14) [inline src/ATen/native/xpu/sycl/ActivationPreluKernels.cpp:7]: Please remove the include if no need.

File: src/ATen/native/xpu/sycl/BinaryShiftOpsKernels.cpp [hunk 7]
```diff
@@ -0,0 +1,48 @@
+#include <ATen/ATen.h>
+#include <ATen/Dispatch.h>
+#include <ATen/native/BinaryOps.h>
```
Comment (fengyuan14) [inline src/ATen/native/xpu/sycl/BinaryShiftOpsKernels.cpp:3]: Please remove the include if no need.

**Comment by fengyuan14** [comment]
Please check if these two operators in op_db cover the native operators in the PR. If so, please add them in `test/xpu/xpu_test_utils.py`,
image
Please add `nn.functional.prelu` in `test/xpu/xpu_test_utils.py`.

File: src/ATen/native/xpu/Activation.cpp (hunk 2 above)
Comment (yucai-intel) [reply src/ATen/native/xpu/Activation.cpp]: Done.

**Review by fengyuan14 — APPROVED** [review]

**Pull request merged**